.gitignore

```diff
@@ -1,2 +1,6 @@
 .pdm.toml
 __pypackages__
+.pdm-python
+.envrc
+.env
+*.db
```

Comment:
Could you bring back the trailing new-line?

Reply:
```suggestion
*.db

```

Reply:
done!